draft-tulshibagwale-saag-pushpull-delivery.md

```diff
@@ -307,14 +311,10 @@ During the Websocket handshake, the Initiator MUST include the value `pushpull`
 ## Verifying the Responder
 The Initiator MUST verify the identity of the Responder by validating the TLS certification presented by the Responder, and verifying that it is the intended recipient of the request, before sending the Communication Object {{communication-object}}.
 
-The Initiator MUST attempt to obtain the OAuth Protected Resource Metadata {{OPRM}} for the Responder endpoint. If such metadata is found, the Initiator MUST obtain an access token using the metadata. If no such metadata is found, then the Initiator MAY use any means to authorize itself to the Responder.
-
 ## Verifying the Initiator
-The Responder MUST verify the identity and authorization of the Initiator. The Responder MAY use common authentication schemes such as Mutual TLS (MTLS) to verify the authenticity of the Initiator. 
-
-Alternatively, the Responder MAY provide OAuth Protected Resource Metadata {{OPRM}} to enable Initiators to obtain appropriate OAuth tokens to authenticate themselves and prove their authorization.
+The Responder MUST verify the identity and authorization of the Initiator. The mechanism by which the Responder verifies the Initiator is out of scope of this specification, but may include mechanisms such as Mutual TLS (MTLS) client authentication, or OAuth access tokens.
 
-Finally, the Responder MAY use other means to authenticate and authorize the Initiator, which are beyond the scope of this specification.
+The Initiator MUST attempt to obtain the OAuth Protected Resource Metadata {{OPRM}} for the Responder endpoint. If such metadata is found, the Initiator MUST obtain an access token using an OAuth authorization server discovered in the metadata. If no such metadata is found, then the mechanism of authenticating to the Responder is out of scope of this specification.
```

Comment:
I moved this here because it's actually talking about what the Initiator sends in order to authenticate to the Responder.